Makefile

```diff
@@ -45,8 +45,11 @@ layers: docker-images
 			echo "### Exporting $${dir} PHP$${php_version}"; \
 			echo "###"; \
 			cd ${PWD} ; rm -rf export/tmp/${layer} || true ; cd export/tmp ; \
-			docker run --entrypoint "tar" bref/$${dir}-php-$${php_version} -ch -C /opt . | tar -x ; \
-			zip --quiet -X --recurse-paths ../`echo "$${dir}-php-$${php_version}" | sed -e "s/layers\//layer-/g"`.zip . ; \
+			CID=$$(docker create --entrypoint=/bin/bash bref/$${dir}-php-$${php_version}) ; \
```

Comment:
I was not very hot to run a container but in fact, chances that cp can fail are nearly close to zero, so :100:

Reply:
We are actually never running the container. We just create it. 

`cp` is not part of the container. It is a docker command. So it will not fail (on docker 1.0 and above.)

Reply:
:warning: ho... but no, in fact... have you tried? Because `scratch` image is trully empty and doesn't contain `bash` at all this souldn't work.

Reply:
This is why I was looking for `docker save` option, which imply to untar each docker layer (tar) to reconstitute the image filesystem, thus to be able to zip it.
This is a little shell script that need to:
1. list all layers (docker save export tar paths in the manifest, just `cat manifest.json | jq -r '.[0].Layers'` to get them).
2. loop over and untar.

Reply:
> We are actually never running the container. We just create it.

ok so this is the beauty of the hack, well done & very subtle :clap:.
So I guess that you can remove `--entrypoint=/bin/bash` and it still work no? :thinking:

Reply:
> ho... but no, in fact... have you tried?

Yes, but please confirm: 

```
make layers layer=blackfire php_versions=74
```

Reply:
> So I guess that you can remove --entrypoint=/bin/bash and it still work no? 🤔

Both yes and no. A container needs an entrypoint to be created. However, we are never using it.. 

I think I've could have done `--entrypoint=/foobar`

Reply:
I've tried and I confirm, it works perfectly! :ok_hand: